Reworks the Twinkleshine-Class

meemofcourse opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request


![2023 05 13-23 20 45](https://github.com/shiptest-ss13/Shiptest/assets/75212565/de6f3a47-7be8-4800-ae73-9fc386e4bf01)
![twinklerework5](https://github.com/shiptest-ss13/Shiptest/assets/75212565/f1808576-70e3-4b56-b977-5b5e7d665fdd)





The Twinkleshine is a CyberSun-made Syndicate display of force, staffed by every branch of the Syndicate. Despite the silly name, the presence of one in a sector implies it to be of importance to the Syndicate, and enemies within sight can only pray that the Twinkleshine crew are incompetent enough to shoot themselves with their own weaponry (or blow themselves up with the supermatter on-board).

It is staffed by:

- 1 Captain
- 1 Lieutenant (previously the Operative - serves as a warden/hos)
- 2 Medics
- 2 Engineers (previously the Mechanics)
- 5 Operatives (previously the Troopers)
- 1 Bartender
- 1 Miner
- 2 Deck Assistants

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Few days ago, an admin spawned a Twinkleshine, and I got to captain it. The Twinkleshine is old. It sucks. This, hopefully, fixes that.

Originally, this was going to be minor fixes, but ended up becoming an attempt at reworking the ship to a more modern state - the hull has been redone and is mostly symmetrical, the old spacepod was replaced with a Dark Gygax, the supermatter shouldn't be activated upon spawning the ship, there's more turf decals and a bigger lot-of-things, added a bartender and a miner, people can actually tell who they are and what they do, and there is now a box moth. Rejoice.

Also, this is the first time I've ever mapped a ship. What better way to begin with a giant battleship?

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
tweak: Reworks the Twinkleshine
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

meemofcourse committed:
adds stuff

like the ship

meemofcourse committed:
minor decaling, adds The Suspicious Burger

meemofcourse committed:
better Kitchen

just adds a freezer on the wall. it looked sad and empty

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

rye-rice reviewed (approved):
thank you so much for keeping the twinkleshine up to par

meemofcourse commented:
mam.p

meemofcourse committed:
Does He Know?

changes a single variable

meemofcourse committed:
Merge branch 'stop-calling-it-a-twink-engine' of https://github.com/meemofcourse/Shiptest into stop-calling-it-a-twink-engine

meemofcourse committed:
fixes another mistyped variable

how many of these do i have

Zevotech commented:
![image](https://user-images.githubusercontent.com/95449138/224400001-7cd8dea5-87b1-4d02-9f88-2fd7c5868566.png)

meemofcourse commented:
wall moth! wall moth! wall moth!

meemofcourse committed:
airlock update

apparently they suck

spockye reviewed (changes requested):
![image](https://user-images.githubusercontent.com/79304582/224485863-e1a78861-1fd5-427e-b14a-c54fb6598294.png)
vendor is a bit akwardly placed. perhaps push forward a bit? 

![image](https://user-images.githubusercontent.com/79304582/224485907-9b1ad66b-7e7d-4d09-93c6-b46e1f53254e.png)
one bed blocks cryo, another is trapped into the corner. Move the one infront of cryo to the side. Do also give them some more normal backpacks and headsets. (also wall light)

![image](https://user-images.githubusercontent.com/79304582/224485962-cb06e559-2e70-4f51-8486-e7fc8ea9ffb8.png)
The cargo bay could use a rework.  The holofans also aren't connected to power


![image](https://user-images.githubusercontent.com/79304582/224486021-f6410054-188d-4273-b73e-025035e902f9.png)
try to open this up a bit. it's way too compact
im also not a huge fan of how it looks. The decalling is a bit too repeating, and it could use some more touches

![image](https://user-images.githubusercontent.com/79304582/224486303-caed9fa0-733c-4ff5-960f-a4a25ce5b800.png)
Not a fan of them having a mech, seeing the already insane amount of guns they have in the armoury

![image](https://user-images.githubusercontent.com/79304582/224486999-c859f894-af9d-46fc-81aa-dbaabacd8ced.png)
missing wall. 

![image](https://user-images.githubusercontent.com/79304582/224486448-fe496c7b-cfc5-4d76-9056-f163fa9b2374.png)
Piping errors

![image](https://user-images.githubusercontent.com/79304582/224486539-b07435ce-c5dd-4ee2-8856-5c13e6e8f76c.png)
Cable errors. Dorms and cargo lack an APC

Captains locker has two 367 loaders, but no gun. 

the telecomms computer in comms is pixelshifted, putting it under the other computer

i'd appreciate if some of the plasma enginers were replaced with ion

the buttons for the launch bays seem switched around?


![image](https://user-images.githubusercontent.com/79304582/224486089-f8563a27-7ad8-4fca-9d30-3ff1561f1aae.png)
locker in each corner, blocking you from going around without opening them. why is there a coffee vendor in the armoury? Wardens locker is kinda weirdly placed, leaving a empty corner.
im not a fan of the armoury. it feels cramped, while still having a lot of open spaces, and a basically complete lack of decalling. Give the windoors access "1" so people without id's cant open it. the wardens office also needs access restrictions

---
All in all its definitely an upgrade, but i do think theres a lot that still needs changing.
Good work!

meemofcourse commented:
ah, so i _do_ suck at mapping
> Captains locker has two 367 loaders, but no gun.

the captain starts with a gun by default. so do the assistants (i'll change that later) and the lieutenant
> Not a fan of them having a mech, seeing the already insane amount of guns they have in the armoury

i'll admit, having them start with a mauler is overkill, but this was meant as a replacement for the space pod they had in the hangar. if you have any suggestions as to what it could be replaced with (or if i shouldn't bother) do tell

Ms-Mee commented:
A Ripley painted black WOULD be kinda funny

meemofcourse commented:
that does sound funny

meemofcourse commented:
![imagen](https://user-images.githubusercontent.com/75212565/224495915-4a59a2dc-ef03-443e-9159-4b46fab4ce3d.png)
literally 1984

meemofcourse committed:
the Review is Stupid I Hate It When My Mistakes are Pointed Out

fixes piping, wiring, apcs, adds more decaling to things, adds boxes and mining gear to cargo, and whatever the fuck else i'm forgetting to put here

meemofcourse committed:
review pt2

adds more decaling, moves the EBRs to the vendor (so you can only get them with a voucher), adds more stuff to the armory, brig lockers, replaces the mauler with a DEATH-RIPLEY (fake one), some other silly stuff like renaming doors

meemofcourse committed:
Door (hopefully not) Stuck

changes access, outfits, a few paths and further improvements to shim. P

meemofcourse committed:
oops

meemofcourse committed:
dust and pipes

meemofcourse commented:
clicky button

Apogee-dev commented:
You know, if a Mauler is overkill (and it is) and a Ripley is underwhelming (which it is) you could give them a Dark Gygax instead. Still a combat mech but not completely ridiculous.

meemofcourse commented:
that sounds like a good in-between

meemofcourse committed:
that testing session really sucked

adds """""better""""" piping near the sm,  the engines actually have cable underneath, adds an apc to dorms (how the fuck did i miss that) adds a obvious fake disk and a manual to the conference table, replaces the DEATH-RIPLEY with a dark gygax so people can continue to weep about not getting a mauler, gives the warden a syndie beret

meemofcourse committed:
what do you do when you run out of funny commit names

fixes invalid variable and some access issues gnbbbggbgg

meemofcourse committed:
i don't even remember what this changes

meemofcourse committed:
better outfits, you can speak in the command channel now

meemofcourse committed:
forgot two commas

pitaden commented:
If your initial clothes get destroyed, the ship doesn't really have spare clothes/shoes for you to grab

Erikafox commented:
@spockye remember to lift / approve once your changes are addressed

Erikafox reviewed (changes requested):
![image](https://user-images.githubusercontent.com/94164348/224785842-a6b6cece-ae00-471e-976e-0f65deef4a98.png)
You need at least 2 high volume dual port air vents for an airlock of this size.
I would also /consider/ adding an interior turret. It is a battlecruiser after all.
![StrongDMM_sO2u0njUpM](https://user-images.githubusercontent.com/94164348/224786107-a8a85dad-3535-4161-8ef3-369818f42d41.png)
these won't layer over the catwalks, so it'll end up looking like shit. 
![image](https://user-images.githubusercontent.com/94164348/224786227-bddeb198-ac40-491f-988d-b86cd81f6656.png)
bad spot for these lockers. 
![StrongDMM_kum9b71beg](https://user-images.githubusercontent.com/94164348/224786294-2ea65904-b032-4345-9619-0f323c62e713.png)
The doublethick decalling won't look /great/, I'd remove the interior line (and the one running on the circuit tiles too)
![StrongDMM_mQ9ZwqHJpB](https://user-images.githubusercontent.com/94164348/224786585-4428c6f7-1c1d-451d-8a0c-c614b93a0db3.png)
same decal layering thing here
![StrongDMM_AlBVX8nyXL](https://user-images.githubusercontent.com/94164348/224786702-b6356036-d82f-4c0a-9e27-ba6b070c03d7.png)
wall moth is hilarious, but will cause the walls to render weirdly. It should go.
![StrongDMM_TaKClmY89V](https://user-images.githubusercontent.com/94164348/224786900-7a8d88da-16a2-493d-a7c4-d7207c429886.png)
Give the mech mech ions or a mech jetpack, otherwise it's going to suck too much ass in space.
![StrongDMM_XmtvkkXmiG](https://user-images.githubusercontent.com/94164348/224787020-e17f58e3-986b-4f7e-9395-f9741270d2f5.png)
move the blast door/pike back by 1 and consider adding some railings/greebling (look at the heron pr's mech launcher) to the spikes.


There's no captain's room like there was before, and I have a few interior design choices I would like to go over with you. Ping me in the discord when you're ready

meemofcourse commented:
fuck

meemofcourse committed:
moves outfits to appropiate datums + review things

most notably remaps the armory, destroys the wallmoth and replaces it with a boxmoth, jobs are actually named properly now so medics can use the crew monitoring + pinpointer, piping, map changes, warcrimes, what more could one ask for

meemofcourse committed:
this is just like george orwell's book

literally 1984. also it removes the funny wire room

meemofcourse committed:
nanotra(n)sen

spare id's description changed. also moves a wallmount so it isn't double wallmount

meemofcourse committed:
what do you MEAN i didn't change it

forgot to press ctrl s

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

meemofcourse committed:
room underneath the captain's room is empty

the room underneath the captain's room is now a janitorial closet, so moved the mop from the engi office to the closet. also adds  a wallsafe with intel, moves scrubbers and wiring around so it makes More Sense, gives the crew a sleepy time box, some other small stuff that made me angry and i forgot to list here. also supervisor lizrd

meemofcourse committed:
Merge branch 'stop-calling-it-a-twink-engine' of https://github.com/meemofcourse/Shiptest into stop-calling-it-a-twink-engine

meemofcourse committed:
airlock update

replaces the break rooms with airlocks (they were getting deconstructed for the table's materials anyways), moves the docking port, changes the spare captain id, adds access restrictions to ids and cards, other misc stuff

meemofcourse committed:
updates outfit backpacks, minor greebling

kitchen and outfit improvements and stuff

meemofcourse committed:
turrets!

plus some more touching up and makes it so the outfits grant faction alingement to the syndicate. this means you won't get shot with the turrets but syndicate hostilemobs might not attack

meemofcourse committed:
no more simplemob teaming

changes Syndicate to PlayerSyndicate. you DON'T get to greet syndicate friends, sorry!

meemofcourse committed:
forgot the fuckin fish

the fish is now Allied

meemofcourse committed:
boxmoth too for good measure

meemofcourse committed:
modifies mech equipment a bit

meemofcourse committed:
hngngngnngngnggnngng

meemofcourse committed:
minor Greebling and Stuff

meemofcourse commented:
![cooltext432286312848364](https://user-images.githubusercontent.com/75212565/227221830-ff118146-5f3d-4ebd-af00-a51381934f8f.gif)

Erikafox commented:
the next review is watching it get played either friday or saturday.

meemofcourse committed:
quickfix

spockye reviewed (changes requested):
![image](https://user-images.githubusercontent.com/79304582/227638267-7c575002-d2cd-470d-9e0f-1c1afe33d176.png)
cable errors

![image](https://user-images.githubusercontent.com/79304582/227638326-97576154-472d-4bd6-9659-9e4cd1165891.png)
still haven't fixed this

![image](https://user-images.githubusercontent.com/79304582/227638570-8ad0dd5b-2ed4-442a-86b9-f84396d64c2b.png)
more cable errors

![image](https://user-images.githubusercontent.com/79304582/227638740-187193ab-150e-4836-bad7-1de132e09700.png)
please switch the tiles outside with the airless subtype

![image](https://user-images.githubusercontent.com/79304582/227639106-2e1b40c1-f150-4086-9899-f9391b6e17b6.png)
piping errors

![image](https://user-images.githubusercontent.com/79304582/227639386-ef7871cd-281a-489f-830b-4dd43d067254.png)
piping errors

![image](https://user-images.githubusercontent.com/79304582/227639496-28b85bcf-1d87-4d1e-a90f-3e97846b3009.png)
piping error

![image](https://user-images.githubusercontent.com/79304582/227639945-2adc689a-a1b8-490e-a0f5-3eee3cc0a51d.png)
no need for wall pipes here!

meemofcourse commented:
d

goober3 commented:
![image](https://user-images.githubusercontent.com/118859017/227643670-0f17f407-5bcc-41f5-8720-63e49b3ac7c7.png)
bad prechargers!

![image](https://user-images.githubusercontent.com/118859017/227653376-72f22371-bb16-43dc-a667-ac7df0939ac2.png)
u should make the floor inside of the sidings different. for Texture. in my opinion

![image](https://user-images.githubusercontent.com/118859017/227653472-bd24db91-5361-42c1-9064-0153d0852e9f.png)
replace the atmos tech locker with an engineer locker. and put an RPD in both of them. both engineers need RPDs for engineering tomfoolery. also consider: Engineering Hardsuits 

![image](https://user-images.githubusercontent.com/118859017/227653868-4f0f72b2-f5be-49b0-8e39-950f690fec8f.png)
![image](https://user-images.githubusercontent.com/118859017/227653892-bc5d373b-6e52-4c75-95db-699a6e26b9ef.png)
maybe find a way to get rid of these Doors. they break up the structure of the ship a little. absolutely not your fault but the engineering department probably shouldn't have access to the armory. especially now that the ship has AACs on the front, too

I think it would be better for this ship to use normal IDs. operative IDs are nice but having a bank account is nicer.

AverageUser67 commented:
ADD A FUCKING AUTOLATHE

meemofcourse commented:
SHUT THE HELL UP I DID THAT ALREADY!!!!!!

BogCreature commented:
The engineering and captain IDs don't have access to their respective departments, meaning you can't download apps with them via the app downloading thingy

Zevotech commented on _maps/shuttles/shiptest/syndicate_twinkleshine.dmm:
```diff
@@ -1,7754 +1,9810 @@
+"wR" = (
+/obj/item/gun/energy/ionrifle,
+/obj/item/gun/ballistic/rocketlauncher/unrestricted{
+	spawnwithmagazine = 0
```
REMOVE THIS! Spawnwithmagazine removes the magazine of the weapon itself instead of the ammo, which results in the PML 9 spawning without an internal magazine and being unable to be loaded.

meemofcourse commented:
god

meemofcourse commented:
> The engineering and captain IDs don't have access to their respective departments, meaning you can't download apps with them via the app downloading thingy

![imagen](https://user-images.githubusercontent.com/75212565/227722342-595e4ca3-6487-4579-b0b3-aaee1a4887eb.png)
i think you just suck. the engi id can't download CIMS though you're right

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

meemofcourse committed:
best mapper to ever exist

fixes. fixes. fixes. oh also replaces the atmos locker with an engi locker and gives engis the ability to download cims

meemofcourse committed:
Merge branch 'stop-calling-it-a-twink-engine' of https://github.com/meemofcourse/Shiptest into stop-calling-it-a-twink-engine

meemofcourse commented:
> I think it would be better for this ship to use normal IDs. operative IDs are nice but having a bank account is nicer.

you have an account id in your notes you can just link it if you alt click your id card. i'll try to automate it maybe

meemofcourse committed:
i don't remember what this did but it was like. cool

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

meemofcourse committed:
leg

meemofcourse committed:
leg?

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

Signed-off-by: meemofcourse <75212565+meemofcourse@users.noreply.github.com>

Erikafox commented:
![image](https://user-images.githubusercontent.com/94164348/229816501-16ae7fdc-a1e1-41b3-a05c-2bf7fa1d71cc.png)
Consider swapping the position of the hall and the quarters here. It should avoid radiation poisoning, because I keep seeing the rooms starting to get green.

meemofcourse committed:
old days (2002 remaster)

exchanges position of hallway and captain's quarters/closet

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

meemofcourse commented:
![imagen](https://user-images.githubusercontent.com/75212565/230212261-b3e4c896-9c40-4ad5-ae58-82fd63892f13.png)
ok there you go

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

Erikafox commented:
![image](https://user-images.githubusercontent.com/94164348/231560873-72515ffb-e43b-4d7d-b339-70263b00a768.png)
Hull shutter

Erikafox commented:
![dreamseeker_JPlrOkqaSp](https://user-images.githubusercontent.com/94164348/231561037-5022a409-e18d-4be1-9ea8-d837d31c0a32.png)
this is in engineering for some reason

Erikafox commented:
![dreamseeker_UzhBlqaVrR](https://user-images.githubusercontent.com/94164348/231562060-0ba98f62-6844-4a71-8bd6-744b6178ed7a.png)
this corner in med is a smidge dark

meemofcourse commented:
> ![dreamseeker_JPlrOkqaSp](https://user-images.githubusercontent.com/94164348/231561037-5022a409-e18d-4be1-9ea8-d837d31c0a32.png) this is in engineering for some reason

noted. messed it up with fixing a merge conflict

meemofcourse committed:
such good mapper

adds lights to medbay, places the shutters where they're supposed to be, removes duplicate organs in medical

meemofcourse committed:
updates op and engi outfit

goober3 commented:
![image](https://user-images.githubusercontent.com/118859017/231574377-6e0ef18f-b83c-4c0d-b5f6-214f5035de40.png)
this Turret is still shooting at the engines 

![image](https://user-images.githubusercontent.com/118859017/231602375-1780551e-dc84-4368-8a23-9381f6881d93.png)
you could swap this console and the operating table around so doctors can perform surgery on two people at once

engineers still get inducers in their belt, this is bad!
an outpost console would be nice

goober3 commented:
oh, and holofans are needed

Erikafox commented:
![image](https://user-images.githubusercontent.com/94164348/231604494-0f203b01-2a6b-4581-910e-72aa6bf12c2e.png)
upper button doesnt work

goober3 commented:
> ![image](https://user-images.githubusercontent.com/94164348/231604494-0f203b01-2a6b-4581-910e-72aa6bf12c2e.png) upper button doesnt work

I think there are quite a few buttons on this ship that don't work. worth checking over all of them

meemofcourse commented:
probably!

meemofcourse committed:
wow i'm so good at mappign guys

fixes buttons

meemofcourse committed:
very important commit

changes wall moth to box moth

Valorium-RB commented:
> ![imagen](https://user-images.githubusercontent.com/75212565/230212261-b3e4c896-9c40-4ad5-ae58-82fd63892f13.png)
> ok there you go

Lizard wine...?

meemofcourse commented:
> Lizard wine...?

actually that's genuine moth wine, made from genuine moth wings, harvested from genuine box twink moths. do you have a problem with it?

spockye reviewed (dismissed):
![image](https://user-images.githubusercontent.com/79304582/232342511-683d8e56-70ec-417e-a44d-af81bc5f553a.png)

i dislike the plants clipping into the lights. please move the plants, or lights

![image](https://user-images.githubusercontent.com/79304582/232342587-fecc972e-ff39-4968-ac93-cda55e383fbd.png)

missing pipes

![image](https://user-images.githubusercontent.com/79304582/232342631-31e104a5-67c3-42f2-8c61-3e137cc76849.png)

missing wires

![image](https://user-images.githubusercontent.com/79304582/232342654-2b74d7f3-0252-4084-ae99-d65785f83051.png)

accidental floor?

![image](https://user-images.githubusercontent.com/79304582/232342804-79767820-e4d4-4d49-843f-1995e7051a82.png)

move the light

![image](https://user-images.githubusercontent.com/79304582/232342860-c457ebb2-c67b-4d2d-9139-c25cdefb408a.png)

move the light (try to avoid having lights in front of wallmounts)

![image](https://user-images.githubusercontent.com/79304582/232343041-dff519fe-3141-4f45-985c-e9605b88cf3c.png)

try to keep yer wall lockers on +-28

![image](https://user-images.githubusercontent.com/79304582/232343279-ceda388d-8cc2-4d3a-a09c-42220f9268b8.png)

this crate blocks the table

![image](https://user-images.githubusercontent.com/79304582/232343320-3bb75031-a7a0-4876-8619-3b5d46cf13df.png)

dont have to, but you could probably just make this a subtype

spockye reviewed (commented):
![image](https://user-images.githubusercontent.com/79304582/232343501-3ce5ea95-9aca-4120-9155-eebda7e5c8ee.png)
(i do still think the mech could be ditched)

meemofcourse commented:
> (i do still think the mech could be ditched)

then what do i replace the room with? :(

spockye commented:
> then what do i replace the room with? :(

same as on the other side, perhaps broken?

meemofcourse commented:
right, but what would I replace the mech with? seeing as how the spacepod the original housed is gone, they still need a funky Battlecruiser weapon

meemofcourse committed:
fixes

Zevotech commented:
> right, but what would I replace the mech with? seeing as how the spacepod the original housed is gone, they still need a funky Battlecruiser weapon

give them ATV's. SYNDICATE ATV'S.

meemofcourse commented:
> > right, but what would I replace the mech with? seeing as how the spacepod the original housed is gone, they still need a funky Battlecruiser weapon
> 
> give them ATV's. SYNDICATE ATV'S.

ok so apparently there's an atv subtype that spawns a turret and puts it on the atv. it does not work when spawned via a ship though so no dice

Erikafox commented:
Leave the mech it's an admin ship. Good setpiece.

spockye reviewed (commented):
![image](https://user-images.githubusercontent.com/79304582/233843509-acbfa449-be3e-4af4-acec-c2f876732fb1.png)

missing wire (engines)

![image](https://user-images.githubusercontent.com/79304582/233843564-7674b445-49c2-4d64-8b82-2f66930f029b.png)

pipe fuckery?

----


Looks good!

meemofcourse commented:
> pipe fuckery?

intentional, leaving it up to engineers to their own devices with it since the sm doesn't produce nitrogen

meemofcourse committed:
fixes errors in review

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

meemofcourse commented:
also, added the mech back

Erikafox reviewed (changes requested):
This needs at least 1 external stationary docking port. Ping me in ruinyard or look at the Borea if you need an example

meemofcourse commented:
damn pirates

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

Signed-off-by: meemofcourse <75212565+meemofcourse@users.noreply.github.com>

meemofcourse committed:
docking. portstssfdsdfv

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

meemofcourse committed:
oops

meemofcourse committed:
adds special twinkle codenames, expands bridge

-id cards now draw from a list of shades of red, randomly pick a number, and put them together so you get a codename. squidtest real. also eveyone spawns with chameleon masks. if someone asks i will change this
-added some utility stuff like fire/emergency/extinguisher cabinets and relocated some items
-added spare id cards
-engineering is now only accessible to engineers
-expanded bridge. ripped a page out of the ancon for this one
-added cameras
-misc greebling

meemofcourse committed:
Merge branch 'shiptest-ss13:master' into stop-calling-it-a-twink-engine

meemofcourse committed:
oops

meemofcourse committed:
docking port stuff

deletes catwalks outside too

meemofcourse committed:
wall insulation

why was it set to 0.1? who knows

meemofcourse committed:
bartender, miner and misc improvements

-adds bartender
-adds miner and appropiate locker to engineering
-updates uniforms to represent syndicate branches (Captain, Lt, Op are ACLF, Med is CyberSun, Eng is GEC, Miner is SUNS/Gorlex, Bartdr is Donk and Assistant is a wildcard)
-adds a donksoft vendor
-rearranges the armory a bit and removes EBR from vendors
-removes an entrance to engineering and replaces it with an airlock
-adds ORM (for miner)
-misc greebling
-some other stuff im probably forgetting

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

meemofcourse committed:
slight outfit update

-moves medic to brig phys (red hud icon cool)
-makes engineer osha compliant
-changes operative and lt outfit

meemofcourse committed:
oops

Erikafox commented:
I was gonna merge this but your int checks are failing.

meemofcourse commented:
oh, that's because i moved doctor to brig phys. i would recommend testing it at least once before merging however since i changed a moderate amount of stuff

Erikafox commented:
sure we can do that wednesday but also fix your integration checks

meemofcourse committed:
oops 2

meemofcourse committed:
Merge branch 'master' into stop-calling-it-a-twink-engine

meemofcourse committed:
twinkleshine final final for real final really final now

-adds medical belts to brig phys and ship (can't you just get them from the vendor?)
-adds a pda for the captain
-adds wiring to cryo apc

meemofcourse commented:
![merge my fucking pr](https://cdn.discordapp.com/attachments/837744059291533395/850099490622799902/merge_my_pr.gif)

Erikafox commented:
yeah sure

meemofcourse commented:
@spockye @Bjarl close ur reviews
